runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -392,6 +414,16 @@ void setPoints(int points)
 		this.points = points;
 	}
 
+	public Instant getInfoTimer()
+	{
+		return infoTimer;
+	}
+
+	void setInfoTimer(Instant infoTimer)
```

Comment:
Same goes for this setter. Also since we use lombok, you can just annotate the field with `@Getter` and `@Setter` and it will generate those methods itself.